TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/logic/instantiation.py

```diff
@@ -37,3 +39,10 @@ def __init__(self, config_file: str):
             self.config.get("metrics_source")
         )
         logger.debug(f"Metrics source initialized: {self.metrics_source}")
+
+        if self.config.get('opensearch_ingestion'):
+            self.osi_migration = OpenSearchIngestionMigration(config=self.config.get('opensearch_ingestion'),
```

Comment:
to add on to the above, I think this should be `self.backfill`, and we should have a function like the one in `models/metrics_source` where we check the `type` parameter to figure out what subclass to instantiate (in this case `OpensearchIngestionBackfill`).

I think all Backfills will need the source and target cluster details, so the `get_backfill_mechanism` function that instantiates the subclasses will probably need those too.

Reply:
Have adjusted now to use `self.backfill`